src/View/Components/_formCrime.html.twig

```diff
@@ -2,27 +2,27 @@
     <div>
 
         <label for="name">Nom :</label>
-        <input type="text" value="{{crimeScene.name}}" name="name">
+        <input type="text" required="required"value="{{crimeScene.name}}" name="name">
```

Comment:
just required is enough ;)

    <input type="text" required value="{{crimeScene.name}}" name="name">